### ISSUE TITLE
pipe.clear_lora() 无效

### ISSUE THREAD

**Issue by XIONGPEILIN** [modelscope/DiffSynth-Studio#996]
import torch
from PIL import Image
from diffsynth.pipelines.qwen_image import QwenImagePipeline, ModelConfig

pipe = QwenImagePipeline.from_pretrained(
    torch_dtype=torch.bfloat16,
    device="cuda:6",
    model_configs=[
        ModelConfig(model_id="Qwen/Qwen-Image-Edit-2509", origin_file_pattern="transformer/diffusion_pytorch_model*.safetensors"),
        ModelConfig(model_id="Qwen/Qwen-Image", origin_file_pattern="text_encoder/model*.safetensors"),
        ModelConfig(model_id="Qwen/Qwen-Image", origin_file_pattern="vae/diffusion_pytorch_model.safetensors"),
    ],
    tokenizer_config=None,
    processor_config=ModelConfig(model_id="Qwen/Qwen-Image-Edit", origin_file_pattern="processor/"),
)

pipe.load_lora(pipe.dit, "train/Qwen-Image-Edit-2509_inpaint_lora/step-5000.safetensors")

pipe.clear_lora()
image = pipe(prompt, edit_image=edit_images, seed=0, num_inference_steps=50, height=512, width=512)
image.save("image_test_1.jpg")
后生成的图片依旧是 load_lora 之后的效果

**Comment by XIONGPEILIN** [comment]
hotload=True 才行
